plugins/ros/src/components/riScPlugin/RiScPlugin.tsx

```diff
@@ -192,6 +211,72 @@ export function RiScPlugin() {
       {riScDialogState !== RiScDialogStates.Closed && (
         <RiScDialog onClose={closeRiScDialog} dialogState={riScDialogState} />
       )}
+      <Dialog
+        open={feedbackOpen}
+        onClose={() => {
+          setFeedbackOpen(false);
+          setFeedbackSent(false);
+        }}
+        fullWidth
+        maxWidth="sm"
+      >
+        {!feedbackSent && <DialogTitle>Send tilbakemelding</DialogTitle>}
+        <DialogContent>
+          {feedbackSent ? (
+            <Typography
+              align="center"
+              variant="h4"
+              sx={{
+                py: 8,
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                height: '100%',
+              }}
+            >
+              Takk for tilbakemeldingen
+            </Typography>
+          ) : (
+            <TextField
+              margin="dense"
+              label="Din tilbakemelding"
+              fullWidth
+              multiline
+              minRows={4}
+              value={feedbackText}
+              onChange={e => setFeedbackText(e.target.value)}
+            />
+          )}
+        </DialogContent>
+        <DialogActions>
```

Comment:
For å få riktig padding:
```suggestion
        <DialogActions sx={dialogActions}>
```